model/feature_flags.go

```diff
@@ -25,9 +25,12 @@ type FeatureFlags struct {
 	// Enable the remote cluster service for shared channels.
 	EnableRemoteClusterService bool
 
-	// AppsEnabled toggle the Apps framework functionalities both in server and client side
+	// Toggle the Apps framework functionalities both in server and client side
 	AppsEnabled bool
```

Comment:
Nit pick:

https://golang.org/doc/effective_go says:

> If every doc comment begins with the name of the item it describes, you can use the doc subcommand of the go tool and run the output through grep. Imagine you couldn't remember the name "Compile" but were looking for the parsing function for regular expressions, so you ran the command,

Should this file follow that standard holistically?

Reply:
@hanzei I've updated the PR to include the flag names in the comments. 

> Should this file follow that standard holistically?

Are you interested in completing this or creating a HW ticket for the change? Alternatively we could just keep the file as-is.

Reply:
I would be in favor of creating a HW ticket if there would be would be a linter to enforce this rules. I thought `golint` had a check for that, but it doesn't. Are you aware of any linter to enforce such rule?